### ISSUE TITLE
Mouse hotkey support?

### ISSUE THREAD

**Issue by iPOSEiDON** [GPUOpen-Tools/ocat#82]
Hello,

I would love to see this having mouse key support,i would preffer having mouse button 4 instead of keyboard as a capture key.

Can it be done ? 

Thank you !

**Comment by aurolou** [comment]
Hi, thanks for the feature request. I see what I can do :)

**Comment by iPOSEiDON** [comment]
Thanks, one question, can a screenshot function be added as well or is it something completely absurd for this tool to have ?

**Comment by aurolou** [comment]
You mean a screenshot of the game you are currently profiling?
I wouldn't say it's completely absurd 🤔 I add it to the potential feature list to track it, and we will discuss it internally.

**Comment by iPOSEiDON** [comment]
Yes, that would be a really nice feature, since this program already has everything else, why not that as well :)
